platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/transients/AbstractTransientDayTimeField.java

```diff
@@ -0,0 +1,22 @@
+package com.softicar.platform.db.runtime.transients;
+
+import com.softicar.platform.common.date.DayTime;
+import com.softicar.platform.db.runtime.table.row.IDbTableRow;
+
+/**
+ * Defines a transient field with values of type {@link DayTime}.
+ * <p>
+ * Please note that null-values are not supported. If the values are optional,
```

Comment:
The Javadoc states that "null-values are not supported" - but yet we use this base class to implement `[Program]QueuedAtField`, which in turn refers to the nullable field `AGProgramState.QUEUED_AT`.

Something's odd or even wrong here.

Reply:
Sorry, it was a copy-and-paste error! I fixed that.